android/app/src/bte/res/values-sk/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You will need to update `app/locales/pull.sh` and `push.sh` to use these new paths and pull into both bte/gps dirs.